main/webapp/modules/core/scripts/views/data-table/data-table-view.js

```diff
@@ -248,7 +249,6 @@ DataTableView.prototype._renderDataTables = function(table, headerTable) {
 
     if (groups.length > 0) {
       var tr = headerTable.insertRow(headerTable.rows.length);
```

Comment:
```suggestion
      var tr = tableHeader.insertRow(tableHeader.rows.length);
```

or just `header` since the table part is obvious.